compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2778,8 +2753,6 @@ fn add_static_crate<'a>(
         let canonical_name = name.replace('-', "_");
         let upstream_rust_objects_already_included =
             are_upstream_rust_objects_already_included(sess);
-        let is_builtins =
-            sess.target.no_builtins || !codegen_results.crate_info.is_no_builtins.contains(&cnum);
```

Comment:
Can the `is_no_builtins` field be removed from `CrateInfo` now?

Reply:
`compiler_builtins` seems to be used at https://github.com/rust-lang/rust/blob/b2d2184edea578109a48ec3d8decbee5948e8f35/compiler/rustc_codegen_ssa/src/back/link.rs#L2614.

Reply:
I see. Fine to leave `compiler_builtins` in that case.